packages/datepickers/src/examples/localization.md

```diff
@@ -0,0 +1,92 @@
+The `<Datepicker>` supports all [date-fns locales](https://date-fns.org/v2.0.0-beta.2/docs/I18n).
+
+```jsx
+const { Field, Label, Hint, Input } = require('@zendeskgarden/react-forms/src');
+const {
+  Dropdown,
+  Menu,
+  Select,
+  Item,
+  Label: DropdownLabel,
+  Field: DropdownField,
+  Hint: DropdownHint
+} = require('@zendeskgarden/react-dropdowns/src');
+const arLocale = require('date-fns/locale/ar-SA');
+const enLocale = require('date-fns/locale/en-US');
+const enGbLocale = require('date-fns/locale/en-GB');
+const esLocale = require('date-fns/locale/es');
+const jaLocale = require('date-fns/locale/ja');
+const zhLocale = require('date-fns/locale/zh-CN');
```

Comment:
I think it is really neat that the locales can just be imported this way. That means I can dynamically import the locale I need.